randovania/data/json_data/prime1.txt

```diff
@@ -1209,21 +1246,29 @@ Asset id: 1423896872
   > Dock to Lower Edge Tunnel
       Trivial
   > Dock to Storage Cave
-      Power Bomb and Grapple Beam and Space Jump Boots and Morph Ball
+      All of the following:
+          Power Bomb and Grapple Beam and Space Jump Boots and Morph Ball
+          X-Ray Visor or Invisible Objects (Beginner)
   > Dock to Security Cave
       Grapple Beam and Space Jump Boots and Morph Ball
 
 > Dock to Storage Cave; Heals? False
   * Plasma Door to Storage Cave/Dock to Phendrana's Edge
   > Dock to Upper Edge Tunnel
       Power Bomb and Morph Ball
+  > Dock to Security Cave
+      All of the following:
+          Power Bomb and Space Jump Boots and Morph Ball
+          Grapple Beam or Standable Terrain (Beginner)
 
 > Dock to Security Cave; Heals? False
   * Morph Ball Door to Security Cave/Dock to Phendrana's Edge
   > Dock to Upper Edge Tunnel
       Morph Ball
   > Dock to Storage Cave
-      Power Bomb and Morph Ball
+      All of the following:
+          Power Bomb and Morph Ball
+          X-Ray Visor or Invisible Objects (Beginner)
```

Comment:
I really don't know/think Xray should be required for this. But i'm not sure.

Reply:
I have it required because someone might forget where the rock wall is, much in the way they might forget the location of power conduits, even though they'd be obvious for a veteran player. That said, it is not marked as a requirement for leaving the cave, as the location of the wall would then be quite obvious.

Reply:
@henriquegemignani thoughts?

Reply:
It seems like a suitable case of Knowledge, or Invisible Objects in this case, in the lowest difficulty possible.
Leaving these tricks off means you can have no idea of where things are and still be fine.

Reply:
Sounds good. I'll go ahead and accept this pr then.